src/test/backslash/readme.md

```diff
@@ -0,0 +1,6 @@
+> see https://aka.ms/autorest
+
+```yaml $(tag) == 'something'
+input-file:
+- specs\some.json
```

Comment:
do we want to allow this ?

Reply:
It looks like `AutoRest` handles this so `Avocado` should as well.